interface/themes/rtl.css

```diff
@@ -300,324 +300,320 @@ text-align:right;
-    float: right !important;
-    transform: rotate(180deg);
+	float: right !important;
+	transform: rotate(180deg);
 }
 
 /* online - portal */
 
 /* patients/summary_pat_portal.php */
 div#portal-buttons-bottom{
-    padding: 5px 15px;
+	padding: 5px 15px;
 }
 
 #DEM .label_custom, #report_parameters table table td.label_custom {
-    text-align: left !important;
+	text-align: left !important;
 }
 
 /* flow board */
 form#pattrk h2>span{
-    float: right !important;
+	float: right !important;
 }
 form#pattrk #inanewwindow>span{
-    float: left !important;
+	float: left !important;
 }
 
 /* cdr Groups */
 .cdr-rules-class .ui-multiselect .ui-state-default {
-    padding: 0 20px;
+	padding: 0 20px;
 }
 
 .rule_detail.edit.summry .left_col {
-    float: none;
-    margin: 5px;
+	float: none;
+	margin: 5px;
 }
 
 .rule_detail .left_col,
 .rule_detail .mid_col,
 .rule_detail .end_col {
-    float: none;
-    display: inline-block;
+	float: none;
+	display: inline-block;
 }
 
 #namecontainer_preminders {
-    float: right !important;
+	float: right !important;
 }
 
 /* Therapy Groups */
 #therapy_groups_list_container #clear_filters{
-    float: right;
+	float: right;
 }
 
 #namecontainer_creminders {
-    float: right !important;
+	float: right !important;
 }
 
 .xdsoft_datepicker .xdsoft_mounthpicker button, .xdsoft_datepicker .xdsoft_mounthpicker > div{
-    float: right;
+	float: right;
 }
 
 .xdsoft_datepicker .xdsoft_mounthpicker button.xdsoft_prev, .xdsoft_datepicker .xdsoft_mounthpicker button.xdsoft_next{
-    -webkit-transform: rotate(180deg);     /* Chrome and other webkit browsers */
-    -moz-transform: rotate(180deg);        /* FF */
-    -o-transform: rotate(180deg);          /* Opera */
-    -ms-transform: rotate(180deg);         /* IE9 */
-    transform: rotate(180deg);             /* W3C compliant browsers */
+	-webkit-transform: rotate(180deg);     /* Chrome and other webkit browsers */
+	-moz-transform: rotate(180deg);        /* FF */
+	-o-transform: rotate(180deg);          /* Opera */
+	-ms-transform: rotate(180deg);         /* IE9 */
+	transform: rotate(180deg);             /* W3C compliant browsers */
 
-    /* IE8 and below */
-    filter: progid:DXImageTransform.Microsoft.Matrix(M11=-1, M12=0, M21=0, M22=-1, DX=0, DY=0, SizingMethod='auto expand');
+	/* IE8 and below */
+	filter: "progid:DXImageTransform.Microsoft.Matrix(M11=-1, M12=0, M21=0, M22=-1, DX=0, DY=0, SizingMethod='auto expand')";
 }
 
 #file_type_whitelist .subject-black-list,
 #file_type_whitelist .subject-white-list,
 #file_type_whitelist .subject-info-save,
 #file_type_whitelist .subject-info-arrows
 {
-    float: right;
+	float: right;
 }
 
 #white-list, #black-list{
-    direction: ltr;
+	direction: ltr;
 }
 
 div.formname{
-    float: right !important;
-    padding-left: 10px !important;
+	float: right !important;
+	padding-left: 10px !important;
 }
-
-.select2-search, .select2-results{
-    direction: rtl !important;
-}
```

Comment:
Just checking if the above 4 lines were missed (as I recall, this was recently added to codebase)